product_docs/docs/ark/3.4/ark_admin/11_ark_securing_ark.mdx

```diff
@@ -57,16 +52,14 @@ Before using the private key, you must modify the permissions on the keyfile. Us
 
 > `chmod 0600 <ssh_key_file>.pem`
 
-Where &lt;ssh\_key\_file&gt;.pem specifies the complete path and name of the EDB Ark ssh private key file.
+Where &lt;ssh_key_file>.pem specifies the complete path and name of the EDB Ark ssh private key file.
```

Comment:
Looks like we're catching `&gt;` but missing `&lt;`

Reply:
Doesn't matter; HTML in markdown has to start with less-than; if it doesn't, it's parsed as Markdown. Ref: [HTML blocks](https://spec.commonmark.org/0.29/#html-blocks), [raw HTML](https://spec.commonmark.org/0.29/#raw-html)

The writer here avoids a lot of unnecessary escapes - note also the lack of backslash escaping on the underscores - this was actually kinda necessary in the original Markdown, but everyone hated it so most parsers will avoid it somehow; CommonMark lays out some sensible rules, but the examples are even better [intraword emphasis examples](https://spec.commonmark.org/0.29/#example-373)